playbooks/wordpress.yml

```diff
@@ -22,15 +22,21 @@
       - php7.3-imap
       - php7.3-json
     - php_mysql_package: php7.3-mysql
+  pre_tasks:
```

Comment:
Nice work using the pre-tasks here. :+1: 
We usually leave all tasks to be run inside the roles, but this is an acceptable one off (we only mount the `www` dir while deploying wordpress.